### ISSUE TITLE
Autocomplete for async variables

### ISSUE THREAD

**Issue by Tiberriver256** [angular/vscode-ng-language-service#255]
## Expected Behavior
When using a template such as:
```
<span *ngFor="let user of users$ | async">{{user.firstName}}</span>
```

I expect firstName to be autocompleted if the property is available on the object.

## Actual Behavior

When using an async pipe it doesn't appear that autocomplete works with any properties on the object.

**Comment by martinkasa** [comment]
It would be sooo useful to have this feature (btw. it works in WebStorm), mostly because async pipe variables are recommended when using observables or ngrx... Is this feature even planned to be implemented?

**Comment by magnattic** [comment]
No feedback for over a year. Is this on the roadmap? This is one of my biggest problems when working with template files.

**Comment by Ploppy3** [comment]
It's not only about auto completion it's also about type & variable checking. It's quite easy to mess things up.

**Comment by ayazhafiz** [comment]
Hi all. Sorry for the delay. @andrius-pra is looking into this now 🙂

**Comment by andrius-pra** [comment]
This issue is already fixed in angular [v9.0.0-next.11](https://github.com/angular/angular/blob/master/CHANGELOG.md#900-next11-2019-10-15)
